src/components/VideoPlayer.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import ReactPlayer from "react-player";
+
+const VideoPlayer = ({ urls }) => {
+    let videoUrl 
+    if (urls) {
+          videoUrl = `https://www.youtube.com/embed/${urls[0].key}`;
+        }
+
+    return <div className="video">
+            {urls && <ReactPlayer url={videoUrl} />}
+        </div>
+} 
+export default VideoPlayer
```

Comment:
Good to have this as a resource since we will need to do this any time we want to implement videos into React